wagtail/admin/static_src/wagtailadmin/scss/_variables.scss

```diff
@@ -48,7 +48,7 @@ $color-grey-5: #fafafa;
 $color-menu-text: #cacaca; // was #aaa wich was too low contrast
 
 $color-thead-bg: $color-grey-5;
-$color-header-bg: $color-teal; // #ff6a58;
+$color-header-bg: $color-teal-darker; // #ff6a58;
```

Comment:
I'm not sure what the comment on this line is referring to, so I didn't change it. The previous color was `#43b1b0`, and the PR changes it to `#358c8b`.